qa-tester/assets/src/js/admin-bar/utils/github.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Base URL for the GitHub API.
+ *
+ * @type {string}
+ */
+const baseUrl = 'https://api.github.com';
+
+/**
+ * Wrapper for `fetch` to catch any errors while making a request.
+ *
+ * @private
+ *
+ * @param {Request|string} resource The resource that you wish to fetch.
+ * @param {Object} opts An object containing any custom settings that you want to apply to the request.
+ * @return {Promise<Response>} A Promise that resolves to a Response object.
+ */
+const _fetch = ( resource, opts = {} ) => {
+	return fetch( resource, opts )
+		.then( ( response ) => {
+			if ( ! response.ok ) {
+				return Promise.reject( response.statusText );
+			}
+
+			return response;
+		} )
+		.catch( ( error ) => {
+			return Promise.reject( error );
+		} );
+};
+
+/**
+ * Get the list of releases.
+ *
+ * @return {Promise<Object[]>} Promise containing list of releases.
+ */
+export const getReleases = () => {
+	/* eslint-disable-next-line prettier/prettier */
+	const url = `${ baseUrl }/repos/ampproject/amp-wp/releases`;
+
+	return _fetch( url ).then( ( response ) => response.json() );
+};
+
+/**
+ * Fetches all open PRs that have build zips available for download.
+ *
+ * @return {Promise<Object[]>} Promise containing a list of PR items.
+ */
+export const getPullRequestsWithBuilds = () => {
+	const url = new URL( `${ baseUrl }/search/issues` );
+	// @todo Should this be limited to open PRs only?
```

Comment:
Yes, this makes sense.

Reply:
If we were to limit it to open PRs we wouldn't be able to access the closed PRs that need to go through QA though.

Reply:
But QA always happens on `develop` or the release branch, so there's no need.